deeplake/api/tests/test_pickle.py

```diff
@@ -53,3 +53,15 @@ def test_dataset(ds):
         np.testing.assert_array_equal(
             ds.label[i].numpy(), unpickled_ds.label[i].numpy()
         )
+
+
+def test_no_chunks_present(local_ds):
```

Comment:
can we rename this test to `test_pickling_dataset_should_not_store_cached_chunks`